lib/internal/modules/esm/translators.js

```diff
@@ -55,6 +55,24 @@ function getSource(url) {
   }
 }
 
+// Use a custom loader getSource hook if one is provided.
+function getSource(loader, url, format) {
+  if (loader._getSource) {
+    return loader._getSource({ url, format }, defaultGetSource);
+  } else {
+    return defaultGetSource(url);
+  }
+}
```

Comment:
If we have already set `_getSource` to the standard value, then this function is unnecessary (saves a few lines / a branch).